digitaltwin/modelrepo/CommonExtensionMethods.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.DigitalTwin.Model.Service
+{
+    internal static class CommonExtensionMethods
+    {
+        private const char ValuePairDelimiter = ';';
+        private const char ValuePairSeparator = '=';
+
+        public static IDictionary<string, string> ToDictionary(this string valuePairString, char kvpDelimiter, char kvpSeparator)
+        {
+            if (string.IsNullOrWhiteSpace(valuePairString))
+            {
+                throw new ArgumentException("Malformed Token");
```

Comment:
"malformed" seems a bit rough, if not innacurate. How about "Missing token"?

Reply:
Missing does not sound right word here, how about incorrect?